### DISCUSSION TITLE
Add option to hold open submenu when clicked on

### DISCUSSION THREAD

**Discussion by kevinfields777** [OpenTTD/OpenTTD#11178] [Features]
Currently when you click on an icon in a menu to select something from a submenu, you have to keep the mouse button hold down in order to access the contents. Could an option be added so that when you click on the menu, the submenu will stay open until you either click on a selection or click on the icon again to close it? This would be great for those who have neurological or dexterity issues and continuing to hold down the mouse button becomes painful or difficult.

**Comment by 2TallTyler** [comment]
https://github.com/OpenTTD/OpenTTD/pull/10938? 😉

**Comment by kevinfields777** [comment]
Aha! Good to know I'm only two months behind the braintrust! :D Thanks!
